src/pykoala/data_container.py

```diff
@@ -73,7 +73,7 @@ def comments(self, comments):
             )
 
     def to_str(self, title=True):
-        """Convert the record into a string."""
+        """Convert the title record into a string."""
```

Comment:
This is wrong. It converts the entire record into a string (i.e. comments + title)